docs/community/index.md

```diff
@@ -12,6 +12,7 @@ collapse: nav
 | Service Name                                                | Creator                                                                                      |
 |-------------------------------------------------------------|----------------------------------------------------------------------------------------------|
 | [RIFT Intel Fusion Tool](./rift-intel-fusion-tool/index.md) | [:octicons-mark-github-16: Nohus](https://github.com/nohus)                                  |
+| [Innominate Appraisal](./innominate-appraisal/index.md) | [:octicons-mark-github-16: Mezoology](https://github.com/mezoology)                                  |
 | [Sample Service](./sample-service/index.md)                 | [:octicons-mark-github-16: Developer McDeveloperface](https://github.com/@your-username-here) |
```

Comment:
Your columns don't match the rest of the table. It still works, but if everyone does their rows differently it will quickly get hard to read.
```suggestion
| [Innominate Appraisal](./innominate-appraisal/index.md)     | [:octicons-mark-github-16: Mezoology](https://github.com/mezoology)                           |
| [Sample Service](./sample-service/index.md)                 | [:octicons-mark-github-16: Developer McDeveloperface](https://github.com/@your-username-here) |
```